project-base/app/src/Model/Transport/Type/TransportTypeData.php

```diff
@@ -7,14 +7,14 @@
 class TransportTypeData
 {
     /**
-     * @var string|null
+     * @var string
      */
-    public $code;
+    public string $code;
 
     /**
-     * @var string[]|null[]
+     * @var array<string, string>
      */
-    public $names;
+    public array $names;
```

Comment:
We do not want to set type hints in Data objects

Reply:
thanks, reverted